wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,207 @@
-| Hold (`3`) | **Left-click (Hold):** Place hold note and adjust length. **Release on end:** Set hold length | Add hold note to current column based on mouse position. |
-
-### Centre (Playfield)
+| `Sampleset` | Auto, Normal, Soft, Drum | Change the sampleset ignoring the current one on timing section. |
+| `Additions` | Auto, Normal, Soft, Drum | Same as above, but overrides the `Sampleset` setting. |
+| `Select` (`1`) | `Left click`**/drag**: Move timestamp and location of a note. `Right click`: Remove note. `Ctrl` **+** `Left click`: Multiple selection. | Select and modify existing notes. |
+| `Circle` (`2`) | `Left click`: Place note. | Add note to current column based on mouse position. |
+| `Hold` (`3`) | **Hold** `Left click`: Place hold note and adjust length. **Release on end**: Set hold end. | Add hold note to current column based on mouse position. |
 
-![osu!mania playfield](/wiki/shared/BEM_PF.jpg "osu!mania playfield")
+### Playfield
 
-**Visual representation of mapping based on current timestamp.**
+![](img/playfield-mania.jpg "Visual representation of the current timestamp")
 
-#### Playfield Boxes
+#### Playfield boxes
 
 | Name | Description |
 | :-- | :-- |
-| Left Box | Notes intensity (Timeline). |
-| Centre | Actual Playfield ([Beat Snap Divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor)). |
+| Left Box | Notes intensity (Timeline) |
+| Centre | Actual playfield ([Beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor)) |
 
-#### Colour notes
+#### Lines and notes colours
 
-| Name | Description |
+| Line colour | Description |
 | :-- | :-- |
-| Double white line | Measure |
-| White line | Common Time |
-| Green line | You / [Judgement](/wiki/Gameplay/Judgement) Line |
-| Blue colour | Currently-selected note |
-| White/Pink/Yellow colour | Normal notes colour |
-
-#### Divisor 1/1
-
-![Divisor 1/1](/wiki/shared/1_1_m.jpg "Divisor 1/1")
-
-- Full Beats
-- White ticks only
-- Best used for Easy difficulty
-
-#### Divisor 1/2
-
-![Divisor 1/2](/wiki/shared/1_2_m.jpg "Divisor 1/2")
-
-- Half Beats
-- Adds red ticks
-
-#### Divisor 1/3
-
-![Divisor 1/3](/wiki/shared/1_3_m.jpg "Divisor 1/3")
-
-- Triplets
-- Has two purple ticks between each white tick
-- Compound Timing
-
-#### Divisor 1/4
-
-![Divisor 1/4](/wiki/shared/1_4_m.jpg "Divisor 1/4")
-
-- Quarter Beats
-- Adds blue ticks, in addition to the red ticks
-- Useful for general mapping
-
-### Centre-Right (Hitsounds & Assist Tools)
-
-![Hitsounds & Assist Tools](img/BE_HSAT.jpg "Hitsounds & Assist Tools")
-
-**List of available subtools complement to the toolkit.**
-
-| Button (Keyboard Shortcut) | Usage | Description |
-| :-- | :-- | :-- |
-| New Combo (`Q`) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse in song script. An acceptable range is about 5-20. |
+| Double white | Full measure |
+| White | Common time |
```

Comment:
this feels off -- what is "common time", even? I think that instead of the whole table it would be better to make a brief reference to the beat snap divisor article, explaining where the line colour come from, and mention the judgement line as below.